### ISSUE TITLE
Types are not published to NPM

### ISSUE THREAD

**Issue by moltar** [junkurihara/js-aws-signature-v4#810]
It is great that this package is written in TypeScript.

But the benefits diminish, because the `.d.ts` files are not published to NPM.

```
❯ tree node_modules/js-aws-sigv4
node_modules/js-aws-sigv4
├── LICENSE
├── README.md
├── dist
│   ├── env.js
│   ├── index.js
│   ├── jsAwsSigv4.bundle.min.js
│   ├── util.js
│   └── v4.js
├── package.json
└── tsconfig.json
```

It would be highly beneficial for TypeScript users to get the types too.

Thanks.

**Comment by junkurihara** [comment]
Hello @moltar 

Thanks a lot for your comment! As you mentioned, in v0.2.17, `d.ts` files were missing. Now I released v0.2.18 on npmjs.com and it includes `d.ts`. 

```
% ls node_modules/js-aws-sigv4/dist 
env.d.ts             index.d.ts           jsAwsSigv4.bundle.js util.js              v4.js
env.js               index.js             util.d.ts            v4.d.ts
```

**Comment by moltar** [comment]
Thanks @junkurihara
